db/migrate/20191118135508_add_language_to_member.rb

```diff
@@ -0,0 +1,5 @@
+class AddLanguageToMember < ActiveRecord::Migration[6.0]
+  def change
+    add_column :members, :language, :integer
```

Comment:
You need a default value here for existing users

Reply:
fixed